packages/shared/src/components/ProfileMenu.tsx

```diff
@@ -128,13 +128,12 @@ export default function ProfileMenu({
         <span className="font-bold typo-title3">{user.name}</span>
         <span className="mt-1 typo-callout">@{user.username}</span>
       </LabeledImage>
-      <div className="flex flex-col -mt-16">
+      <div className="flex relative flex-col -mt-16">
```

Comment:
Not super urgent, but actually these probably shouldn't be `Button` button `button` as they have no overlap styling and just render for the onClicks right?

Reply:
Well, they do have some hover state styling, which actually is the same as tertiary Button variant.